components/brave_federated/data_stores/data_store.cc

```diff
@@ -73,9 +90,61 @@ bool DataStore::Init(int task_id,
   return db_.Open(database_path_) && EnsureTable();
 }
 
-DataStore::~DataStore() {}
+bool DataStore::AddTrainingInstance(
+    const mojom::TrainingInstancePtr training_instance) {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  sql::Statement s1(db_.GetUniqueStatement(
+      base::StringPrintf("SELECT MAX(training_instance_id) FROM %s",
+                         task_name_.c_str())
+          .c_str()));
+  s1.Step();
+  int training_instance_id = s1.ColumnInt(0) + 1;
+
+  base::Time created_at = base::Time::Now();
+
+  for (const auto& covariate : training_instance->covariates) {
+    sql::Statement s(db_.GetUniqueStatement(
```

Comment:
`s` could be `statement` to be more descriptive, see other comment about use of abbreviations

Reply:
wouldn't this be redundant with the type just beside it?

Reply:
not at all as the s is then used elsewhere, and it is against coding style guide to use such abbreviations